Added Unfettered Support to First Contact: Hai (#5336)
endless-sky/endless-sky: data/hai/hai missions.txt

			`	"Yes," it says, "but be careful. The north is the territory of some renegade Hai: bandits and pirates, living off what they steal from the rest of us. If you travel among them, you will not be safe."`
			`	You talk for a while longer, but do not gain much additional information except that several of their worlds are willing to sell Hai technology even to human beings. You thank them both for stopping to talk with you, and the Hai responds, "We are always glad to welcome new friends. May peace dwell in your heart until we meet again."`
				decline
			label unfettered
			`	The Hai here seem more friendly and approachable than those you met earlier. However, the most surprising aspect of the spaceport is not the aliens, but the fact that human merchants and other civilians are walking among them and are clearly at home here. Would you like to find someone who can tell you what is going on here?`
			choice
				`	(Sure.)`
				`	(Not right now.)`
					defer
			`	You walk up to a human merchant who is busy haggling with a Hai. They interrupt their conversation when they see you approaching, and the human gestures for you to join them. "First time down the rabbit hole?" he asks.`

> 			`	You shake your head and explain how you met with the Unfettered that called themselves the 'True Hai'.`

Reviewer: ```suggestion
			`	You shake your head and explain how you met with the Unfettered that called themselves the 'True Hai.'`
```

Author: Nice catch.

			choice
				`	"Why do you allow humans to settle here?"`
				`	"What do you ask for in return for letting humans settle here?"`
			`	"Humans are a young species," the Hai says, "full of energy, full of new ideas. And the Hai are old, and everything we do is what we have done before. When humans go on vacation, they travel to a world with perfect weather, sunny every day. When Hai go on vacation, we visit the stormiest world, to find unpredictability and change. Humans are so strange, that to speak with a human is like a small vacation." It smiles, and you catch a glimpse of its massive incisors. Based on your knowledge of xenobiology you would guess that the Hai are herbivores, but you can't be certain.`
			choice
				`	"So I'm allowed to travel throughout your territory if I want?"`
					goto north
				`	"Can I ask more about the Unfettered now?"`
					goto questions
				`	"Thank you, but I have no more questions."`
					decline
			label questions
			apply
				set "Ask the Hai about the Unfettered: offered"

> 			`	The Hai seems willing to explain more, what would you like to ask?`

Reviewer: You just asked a question, so have the Hai actually respond to it instead of saying "they look like they'll give you an answer."
```suggestion
			`	"What exactly would you like to know?" the Hai responds.`
```

					goto raids
				`	"They said the Drak altered the Hai to make them less aggressive. Is that true?"`
					goto drak
				`	"No, I want to know more about your society. Is it as peaceful as it appears?"`
					goto peaceful
				`	"That's all I wanted to know. Thanks."`
					decline
			label raids
			`	He pauses. "Hundreds of years ago, our territory was raided by aliens from the east, who came in massive warships and plundered our worlds. In those days there were some Hai who were brave enough to risk their lives to defend us. The raids lasted for decades, then abruptly stopped.`
			`	"Most of those who had fought the invaders were glad to return to ordinary life, but there were some for whom life now seemed bland and meaningless without the excitement and chaos of war. They started colonies in the systems to the north, and fought amongst themselves, and mismanaged the land, and grew in numbers. And now their worlds cannot feed them all, and they invade us and tell stories to put blame on us for their failures."`
			choice
				`	"Shouldn't they be treated with gratitude for defending you against the raiders?"`
				`	"I have other questions."`

> 					goto questions

Reviewer: Don't loop back to questions, because otherwise we can very awkwardly ask the same question multiple times. Instead, simply follow the same structure as "Ask the Hai..."

			choice
				`	"I still have more questions."`
					goto questions
				`	"That's all I wanted to know. Thanks."`
					decline			
			label north
			`	"Yes," it says, "but be careful if you return to the north. The Unfettered are dangerous. If you travel among them, you will not be safe."`
			choice
				`	"I would like to ask about the Unfettered"`
					goto questions
				`	"I will be careful"`
			`	You talk for a while longer, but do not gain much additional information except that several of their worlds are willing to sell Hai technology even to human beings. You thank them both for stopping to talk with you, and the Hai responds, "We are always glad to welcome new friends. May peace dwell in your heart until we meet again."`

> 				decline

Reviewer: I'd put this above `label questions`. You can also ditch `label north` and `goto north` if you do that, because the first choice from the `choice` above `label questions` would lead right into it.

Author: I think I did these changes, but I'm not certain I did them right, so leaving these open for now.

			`	You talk for a while longer, but do not gain much additional information except that several of their worlds are willing to sell Hai technology even to human beings. You thank them both for stopping to talk with you, and the Hai responds, "We are always glad to welcome new friends. May peace dwell in your heart until we meet again."`
				decline
			
			label unfettered
			
			`	The Hai here seem more friendly and approachable than those you met earlier. However, the most surprising aspect of the spaceport is not the aliens, but the fact that human merchants and other civilians are walking among them and are clearly at home here. Would you like to find someone who can tell you what is going on here?`
			choice
				`	(Sure.)`
				`	(Not right now.)`
					defer
					
			`	You walk up to a human merchant who is busy haggling with a Hai. They interrupt their conversation when they see you approaching, and the human gestures for you to join them. "First time down the rabbit hole?" he asks.`
			`	You shake your head and explain how you met with the Unfettered that called themselves the 'True Hai'.`

> 			`	The hai says, "Ah, you have met our wayward sisters and brothers to the north. Is that what you have come to ask us?`

Reviewer 2: ```suggestion
			`	The hai says, "Ah, you have met our wayward sisters and brothers to the north. Is that what you have come to ask us about?`
```

Author: Nice catch, that definitely flows better.

> 			`	The hai says, "Ah, you have met our wayward sisters and brothers to the north. Is that what you have come to ask us about?`

Reviewer 3: ```suggestion
			`	The Hai says, "Ah, you have met our wayward sisters and brothers to the north. Is that what you have come to ask us about?`
```

			choice

> 				`	"Yes," you say. "Can you tell me why you are at war?"`

Reviewer 3: ```suggestion
				`	"Yes. Can you tell me why you are at war?"`
```

Author: Considered this one myself, but was uncertain which to go with.

				`	"No, I wanted to know more about your society. Is it as peaceful as it appears?"`
					goto peaceful

> 					

Reviewer 3: I'm pretty sure all these empty extra lines are unnecessary. It seems like there some extra lines scattered throughout this file already, but I think the amount of those lines in this newly-written section is just a little overboard compared to the rest of the game data files. In fact, compared to the section from "Ask Hai about the Unfettered" that most of this is copied from, you've added in quite a few extra empty lines. You could probably tighten this up it a little, but it's a nit-picky thing.

Author: Yeah, I was having trouble following where stuff was breaking, so I needed the extra lines to break up the text, I can remove them all.

			choice
				`	"Why do you allow humans to settle here?"`
				`	"What do you ask for in return for letting humans settle here?"`
				
			`	"Humans are a young species," the Hai says, "full of energy, full of new ideas. And the Hai are old, and everything we do is what we have done before. When humans go on vacation, they travel to a world with perfect weather, sunny every day. When Hai go on vacation, we visit the stormiest world, to find unpredictability and change. Humans are so strange, that to speak with a human is like a small vacation." It smiles, and you catch a glimpse of its massive incisors. Based on your knowledge of xenobiology you would guess that the Hai are herbivores, but you can't be certain.`
			choice
				`	"So I'm allowed to travel throughout your territory if I want?"`
				`	"Can I ask more about the Unfettered now?"`
					goto questions
				`	"Thank you, but I have no more questions."`
					decline
			
			`	"Yes," it says, "but be careful if you return to the north. The Unfettered are dangerous. If you travel among them, you will not be safe."`
			choice

> 				`	"I would like to ask about the Unfettered"`

Reviewer 3: ```suggestion
				`	"I would like to ask about the Unfettered."`
```

					goto questions

> 				`	"I will be careful"`

Reviewer 3: ```suggestion
				`	"I will be careful."`
```
